src/App.js

```diff
@@ -1,18 +1,21 @@
 import { Route, Routes } from "react-router-dom";
 import "./App.css";
-import {Home, LikedVideos, Login, PrivateRoute, SignUp, UserProfile, WatchLater} from "./pages/index"
+import {HistoryPage, Home, LikedVideos, Login, PrivateRoute, SignUp, SingleVideo, UserProfile, WatchLater} from "./pages/index"
 
 function App() {
   return (
     <div className="App">
       <Routes>
         <Route path="/" element={<Home />}/>
+        <Route path="/video" element={<Home />} />
```

Comment:
I suggest it would be better and more self explainatory if you use path="/home" for home component on line 10